data/governments.txt

```diff
@@ -1402,8 +1560,10 @@ government "Remnant"
 	"attitude toward"
 		"Indigenous Lifeform" 0.05
 		"Korath" -.05
+		"Kor Alaji" -.1
 		"Alpha" -.05
 		"Remnant (Research)" 1
+		"Korath (Civilian)" 0.02
```

Comment:
The population figures of korath civilian ships are such that there is a significant difference between 0.01 and 0.02. 

Unless I'm misremembering exactly how it works... For most factions in the game, yes, 0.01 and 0.02 are functionally identical because most ships have tiny crew. The Korath Civilian ships have such high crew, however, that the same sort of hosility or friendship can be obtained with a single ship that might take hundreds of ships with other governments.

Just looking at the wiki, and there's no mention of crew count impacting the values. So perhaps I'm misremembering things.

Reply:
Trawling through the code to fact check the wiki, there is in fact an explicit .05 threshold in order to effect reputation changes.  And yes, crew does affect the reputation change.